README.md

```diff
@@ -230,6 +230,36 @@ sudo systemctl enable et.service
 sudo systemctl start et.service
 ```
 
+### Raspbian (Stretch):
+#### (_tested on_: Raspberry Pi 3B+):
+
++ Install dependencies
+
+	sudo apt install python3-protobuf protobuf-compiler
```

Comment:
I don't think we need python3, installing protobuf-dev should be good enough

Reply:
I was able to get enough dependency by running:
```
sudo apt-get install libprotobuf-dev
```

Reply:
On top of the protobuf deps, I also need to install:
```
sudo apt get libsodium-dev
sudo apt get libevent-pthreads-2.1-6
```

Reply:
My device is Raspberry Pi 4B with Raspbian.